### ISSUE TITLE
do inputs work with react linkState?

### ISSUE THREAD

**Issue by contra** [jhudson8/react-semantic-ui#2]
http://facebook.github.io/react/docs/two-way-binding-helpers.html

**Comment by jhudson8** [comment]
No, I was not aware of linkState when I made these components.  PR's welcome

**Comment by contra** [comment]
Wow - what an extremely hostile attitude. "PR's welcome - closed" wtf.

If this is something you actually wanted people to send a PR for you would leave it open, auto-closing every issue defeats the point of an issue tracker. We had a choice between this and react-semantify and chose this, but with this anti-contributor hostility you're going to put a lot of people off

**Comment by jhudson8** [comment]
whoops, ment to do comment rather than close and comment.  not hostility intended

**Comment by jhudson8** [comment]
Thought about this more and want to apologize again for my initial response.  I was hastily going through issues with some of my projects before running out and, after re-reading my response, it was def dick-ish - not meant to be though.

**Comment by contra** [comment]
@jhudson8 Not a problem :palm_tree:
